tls_codec/src/tls_vec.rs

```
@@ -168,9 +180,10 @@ macro_rules! impl_serialize_common {
         $(#[$std_enabled])?
         fn get_content_lengths(&$self) -> Result<(usize, usize), Error> {
             let tls_serialized_len = $self.tls_serialized_len();
+            hax_lib::assume!(tls_serialized_len > $len_len); // underflow
```

We should return an error here instead for the underflow.